src/assets/gif/index.js

```diff
@@ -0,0 +1,3 @@
+import pagenotfound from './pagenotfound.gif';
+
+export {pagenotfound};
```

Comment:
you don't have to import and then export, you can write like this instead
`export default as { componentName }  from "path/to/file"` if you have exported as default from the component file

Reply:
images are not exported by themselves in the same file, so I've to import them and then export.